tests/checkpoint_transaction/conf/dm-worker1.toml

```diff
@@ -0,0 +1,2 @@
+name = "worker1"
```

Comment:
please add worker2 and source2 into test, and make sure shardddl is working properly

Reply:
I think ddl does not need to be tested, because the code will wait for all ddl to finish executing in `s.jobwg.Wait()`